h2o-core/src/main/java/water/MRTask.java

```diff
@@ -252,7 +252,8 @@ public void map( Chunk c, NewChunk nc ) { }
    *  <strong>local</strong> Chunks.  All map variants are called, but only one
    *  is expected to be overridden. */
   public void map( Chunk c0, Chunk c1 ) { }
-  //public void map( Chunk c0, Chunk c1, NewChunk nc) { }
```

Comment:
Yes, I understand that. But in the code below we actually have code that tries to invoke this functions: `map(bvs[0], bvs[1], appendableChunks[0]);`. Of course, since the function is commented out, it ends up calling `map(Chunk, Chunk, Chunk)` instead (since `newChunk` inherits from `Chunk`), which is obviously a problem...